src/services/notifier/utils/updaterUtils.ts

```diff
@@ -52,17 +55,21 @@ export const updateSubscriptionsBy = async (
 ): Promise<void> => {
   const [host, port] = providerUrl.split(/(?::)(\d*)$/, 2)
   const svcProvider = new NotifierSvcProvider({ host, port })
-  const subscriptionsDTO: any[] = await svcProvider.getSubscriptions(consumerAddress, subsHashesToUpdate)
+  try {
+    const subscriptionsDTO: any[] = await svcProvider.getSubscriptions(consumerAddress, subsHashesToUpdate)
 
-  await Promise.all(subscriptionsDTO.map(({
-    hash,
-    status,
-    paid,
-    notificationBalance
-  }) =>
-    SubscriptionModel.update(
-      { status, paid, notificationBalance },
-      { where: { hash } }
-    )
-  ))
+    await Promise.all(subscriptionsDTO.map(({
+      hash,
+      status,
+      paid,
+      notificationBalance
+    }) =>
+      SubscriptionModel.update(
+        { status, paid, notificationBalance },
+        { where: { hash } }
+      )
+    ))
+  } catch (error) {
+    logger.error('Unable to update subscriptions from notifier service provider', error)
```

Comment:
How about using logger.warn instead of error. Since this is just a warning we are giving by logging the error. It would be easier to distinguish also with actual errors.